setup.py

```diff
@@ -47,6 +47,8 @@
     "black",
     # for %%sql --interact
     "ipywidgets",
+    # for %sqlcmd explore --table
```

Comment:
let's add a more accurate description (at first, I thought this was required for the feature itself, but looks like it's only used for the tests)

```suggestion
    # for running tests for %sqlcmd explore --table
```